exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestEvaluationVisitor.java

```diff
@@ -72,6 +73,37 @@ public void x() throws Exception{
     logger.debug(g.generateAndGet());
   }
 
+  @Test
+  public void testVisitBooleanOrWithoutFunctionsEvaluation() throws Exception {
```

Comment:
I agree that new test cases use higher level, SQL. The first thought was TestExampleQueries class, but this class is overloaded.
TestBugFixes will be better. Thanks.